lava-slave-install.md

```diff
@@ -0,0 +1,114 @@
+This is an howto install a lava-docker slave
+This howto is based on a Debian host.
+
+# Create a dedicated user
+The user will be lavadocker in our example. It must be in the docker group for doing docker operations.
+```
+useradd -G docker lavadocker
+```
+
+# Install all pre-requisites packages
+docker-compose need to be installed from PIP
+```
+apt-get install git python-pip
+```
+
+# Install docker-ce
+See https://docs.docker.com/install/linux/docker-ce/debian/ for more detailled informations.
+## Using docker sources need https and their certificates are signed by ca-cert
+```
+apt-get install apt-transport-https ca-cacert
+```
+
+## Download docker GPG
```

Comment:
You don't need to document this step (and the next couple) because you're already pointing people to the Docker instructions for how to install Docker.  Don't duplicate the docker instructions here.